e2e/utils/functions/bahmni.ts

```diff
@@ -92,10 +103,18 @@ export class Bahmni {
   }
 
   async goToHomePage() {
-    await this.page.goto(`${BAHMNI_URL}/bahmni/home/`);
+    await this.page.goto(`${BAHMNI_URL}/bahmni/home`);
     await expect(this.page).toHaveURL(/.*home/);
   }
 
+  async goToDiagnosis() {
```

Comment:
I find it more meaningful to use the term `navigateTo*()` i.e `navigateToDiagnosis()` here.